Source/iOS/UIColor+Hue.swift

```diff
@@ -173,6 +173,30 @@ public extension UIColor {
       return a
     }
   }
+
+	var hueComponent : CGFloat {
```

Comment:
Can you remove the space before the `:` to match our convention?

Reply:
That's funny because I never leave that space but left it following the coding convention on the same extension you used for redComponent.